test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,23 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(500);
```

Comment:
Hmmm, looks like we didn't do that for AIX yet? https://github.com/nodejs/node/blob/master/test/common.js#L232-L240

In this case I suggest setting the timeout to 500 only if `common.isAix` is `true`.